VideoLocker/res/layout/row_discussion_comment.xml

```diff
@@ -24,26 +24,27 @@
         android:textSize="@dimen/edx_x_small"
         tools:text="This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment." />
 
-    <LinearLayout
+    <FrameLayout
```

Comment:
Using `FrameLayout` is the wrong approach, because it doesn't prevent overlapping. You should go with the initial approach of using a placeholder view that takes over the extra space I guess.